packages/browser/README.md

```diff
@@ -40,33 +42,20 @@ const options: BacktraceConfiguration = {
     url: "https://submit.backtrace.io/<universe>/<token>/json",
 }
 
-// Initialize the client
+// Initialize the client with the options
 const client = BacktraceClient.initialize(options);
 
-// Send an error
+// By default, Backtrace will send an error for Uncaught Exceptions and Unhandled Promise Rejections
+
+// Manually send an error
 client.send(new Error("Something broke!"));
 ```
 
-## Sending an Error
-By default, Backtrace will send an error for Uncaught Exceptions and Unhandled Promise Rejections.
-
-### Manually Sending an Error
-There are several ways to send an error to Backtrace. For more details on the definition of ```client.send()``` see [Methods](#methods) below.
-
-```ts
-// send as a string
-await client.send('This is a string!');
-
-// send as an Error
-await client.send(new Error('This is an Error!'));
-
-// as a BacktraceReport (string)
-await client.send(new BacktraceReport('This is a report with a string!'));
-
-// as a BacktraceReport (Error)
-await client.send(new BacktraceReport(new Error('This is a report with a string!')));
-```
+### Upload source maps
+Client-side error reports are based on minified code. Upload source maps and source code to resolve minified code to your original source identifiers.
 
+[(Source Map feature documentation)](https://docs.saucelabs.com/error-reporting/platform-integrations/source-map/)
+<? TBD: Link to source upload doc ?>
```

Comment:
I believe the PR Sebastian made to the Sauce docs includes source map, processing, and source upload and it will update the linked doc. Unless it gets broken out to separate docs.

Reply:
We will verify after that is merged